kuma/wiki/jinja2/wiki/list/needs_review.html

```diff
@@ -23,13 +23,13 @@
         {% if tag|string=='editorial' %}
           <p>
             {% trans url=wiki_url('MDN/Contribute/Howto/Do_an_editorial_review') + '#core-steps' %}
-            Please review these articles for spelling, grammar, formatting, layout, etc., and then clear the Editorial review flag. <a href="{{ url }}">See more detailed instructions</a>.
+            {{ _('Please review these articles for spelling, grammar, formatting, layout, etc., and then clear the Editorial review flag. <a href="%(url)s">See more detailed instructions</a>.', url=url) }}
```

Comment:
``{% trans %}`` already translates this block of text.  The string is missing from the translation interface because I haven't done string extraction since merging this code. Remove changes to this file from the PR.